### PR TITLE
Some _suggestions_ with the text.

### PR DESCRIPTION
**Pull request by richardneililagan**
Because you'll inevitably get a similar PR for most of these points. :p

### CODE REVIEW

**Comment by jeroenvisser101** [comment]
:+1: thanks for taking time for this!

File: docs/components.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 Components
 ==========
 
-Think in components. Consider of each piece of your UI as an individual "component."
+Think in components. Consider each piece of your UI as an individual "component".
```
Comment (rstacruz) [inline docs/components.md:4]: `"xxx."` is commonly preferred but I guess I'll allow it. http://www.grammarbook.com/punctuation/quotes.asp

**Comment by rstacruz** [comment]
werd!

File: docs/components.md (hunk 1 above)
Comment (richardneililagan) [inline docs/components.md:4]: Hey, hey! TIL! :D

**Pull request merged**